data/tascamgtk.desktop

```diff
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Encoding=UTF-8
+Name=TascamGtk
+Name[fr]=TascamGtk
+GenericName=Tascam US-16x08 DSP mixer application
+GenericName[fr]=contrôle du DSP de la Tascam US-16x08
```

Comment:
In most (if not all) other desktop files, the GenericName start with a capital letter.